atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -110,17 +110,17 @@ public static LocalPaxosServices createInstrumentedLocalServices(
         return createInstrumentedLocalServices(
                 metricsManager,
                 config,
-                runtime,
                 remotePaxosServerSpec,
+                () -> RemotingClientConfig.ALWAYS_USE_LEGACY, // TODO (jkong): Wire this up or change it to Conjure
```

Comment:
Why legacy?

Reply:
As before, this is used by the embedded leader mode, which I fear we haven't tested as well under CJR yet, so would prefer to do this in another PR.